Makefile

```diff
@@ -50,3 +50,12 @@ subsystem: build-image
 
 generate:
 	go generate $(shell go list ./...)
+
+.PHONY: check
+check: go-import go-fmt
+
+go-import:
+	goimports -w -l . || /bin/true
+
+go-fmt:
```

Comment:
goimports also formats the code, so there is not need to perform gofmt

Also later versions of goimports will return non-zero return code if there were formatting issues

Reply:
Done